Description: Fixed path in ksm-wrapper script
 ksm-wrapper tried to search libksm_preload.so in /usr/bin, the error
 message:
 
  Could not find /usr/bin/libksm_preload.so. Have you compiled it with 'make'?
 
 Now it just uses "libksm_preload.so" instead of "/usr/bin/libksm_preload.so".
Author: Dmitry Yu Okunev <dyokunev@ut.mephi.ru>
Forwarded: not-needed
Last-Update: 2017-01-10

--- ksm-preload-0.10.1.orig/scripts/ksm-wrapper
+++ ksm-preload-0.10.1/scripts/ksm-wrapper
@@ -18,8 +18,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-readonly KSM_PATH=$(cd $(dirname $0) ; pwd)
-readonly KSM_SO="${KSM_PATH}/libksm_preload.so"
+readonly KSM_SO="libksm_preload.so"
 
 if [ ! -x "${KSM_SO}" ]; then
     echo "Could not find ${KSM_SO}. Have you compiled it with 'make' ?"
